src/altinn-app-frontend/src/utils/formComponentUtils.ts

```diff
@@ -200,12 +215,31 @@ export const getDisplayFormData = (
       });
       return label;
     }
+    if (isFileUploadComponent(component) || isFileUploadWithTagComponent(component)) {
```

Comment:
Some unit tests for this whole block would be sweet!

Reply:
Done! :tada: